tools/fast/process.go

```diff
@@ -182,6 +185,145 @@ func (f *Filecoin) DumpLastOutputJSON(w io.Writer) {
 	}
 }
 
+// outputMirror is used to store a copy of output lazily
+type outputMirror struct {
```

Comment:
this was added to make `LastOutput` work, correct?

Reply:
yes